src/CMakeLists.txt

```diff
@@ -48,13 +48,13 @@ if (TILES)
                 ${MESSAGES_CPP}
                 ${RESOURCE_RC})
         if(MSYS2)
-            # /MANIFESTINPUT not supported by GNU linker
-            # Defaul embed via `windres.exe` compiling `resource.rc` -> `resource.rc.obj`
+            target_link_options(cataclysm-tiles PRIVATE
+                "$<$<CONFIG:Debug,RelWithDebInfo>:LINKER:--strip-debug>"
```

Comment:
Maybe I misunderstand but does this remove debug info from the debug build? Having the debug info is the very purpose of the debug build so I doubt this is useful to anybody.

> Need `--strip-debug`, otherwise I get:
> 
> ```
> Program 'cataclysm-tiles.exe' failed to run: The specified executable is not a valid application for this OS plat
> ```

If you're getting this from the debug build, a possible reason is that the executable exceeds 4 GB and cannot be loaded by Windows. In that case you can use the following commands to separate the debug symbol from the executable or remove the debug info in your local workflow, but I don't think we should remove that information in the in-repo script.

```bash
objcopy --only-keep-debug --compress-debug-sections cataclysm-tiles.exe cataclysm-tiles.debug
objcopy --strip-unneeded --add-gnu-debuglink=cataclysm-tiles.debug cataclysm-tiles.exe
```

Reply:
Please check #71805